examples/light-switch-app/silabs/src/LightSwitchMgr.cpp

```diff
@@ -26,18 +26,123 @@
 
 #include "AppConfig.h"
 #include "AppEvent.h"
+#include "AppTask.h"
 
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app-common/zap-generated/cluster-objects.h>
 #include <app/clusters/switch-server/switch-server.h>
 #include <platform/CHIPDeviceLayer.h>
+#include <cmsis_os2.h>
+#include <platform/silabs/platformAbstraction/SilabsPlatform.h>
 
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
+using namespace chip::DeviceLayer;
+using namespace chip::DeviceLayer::Silabs;
 
 LightSwitchMgr LightSwitchMgr::sSwitch;
 
+AppEvent CreateNewEvent(AppEvent::AppEventTypes type)
```

Comment:
this shouldn't be a global function - this needs to go in an anounymous namespace

Reply:
Moved this to LightSwitchMgr class